tests/boot/boot_from_pxe.pm

```diff
@@ -50,6 +50,10 @@ sub run {
         $openqa_url = 'http://' . $openqa_url unless $openqa_url =~ /http:\/\//;
         my $repo = $openqa_url . "/assets/repo/${image_name}";
         $image_path = "$path/linux initrd=$path/initrd install=$repo";
+        if (check_var('BACKEND', 'ipmi')) {
+            my $netdevice = get_var('SUT_NETDEVICE') || 'eth0';
```

Comment:
This will always fill $netdevice with 'eth0' - you programmed to much python ;)
IMHO the best is to use the built-in default mechanism of get_var: http://open.qa/api/testapi/#_get_var

Reply:
I don't believe you: http://imagebin.nue.suse.com/2285
;)

But, yes, the built-in mechanism is better.